packages/@ember/engine/index.js

```diff
@@ -417,15 +416,6 @@ Engine.reopenClass({
     return registry;
   },
 
-  /**
-    Set this to provide an alternate class to `DefaultResolver`
-
-    @deprecated Use 'Resolver' instead
-    @property resolver
-    @public
-  */
-  resolver: null,
```

Comment:
This is weird. I see it was added in https://github.com/btecu/ember.js/commit/7b0a7ecb5b9e863bded0094d8b7e5c1861e1d52f#diff-a7583c2d819fdabaff78778f93e5112adc888bc3026338522862f08ddfadb4abR364 (or https://github.com/emberjs/ember.js/pull/12685/files#diff-a7583c2d819fdabaff78778f93e5112adc888bc3026338522862f08ddfadb4abR377 in the PR) by @dgeb but I don't see any usage of it. What am I missing? It was added with a `@deprecated` flag in jsdoc, but I don't see what is firing the deprecation.

Reply:
> but I don't see what is firing the deprecation.

I wonder if the code firing the deprecation was removed prior to 3.0 without removing the deprecated prop? Regardless, it doesn't seem that the `resolver` class prop is referenced, so it seems safe to remove this now.